api/types/events/events.proto

```diff
@@ -1673,6 +1673,26 @@ message WindowsDesktopSessionEnd {
     map<string, string> DesktopLabels = 8 [ (gogoproto.jsontag) = "desktop_labels" ];
 }
 
+message UserCertsEmit {
```

Comment:
Please add documentation explaining what the event is and what causes it to be emitted.

Are events 1:1 to certificates issued? Is it worth including which kinds of certificates were generated in the process? Please document too.